depro-inactive-guests.test.js

```diff
@@ -0,0 +1,40 @@
+const fs = require("fs");
+const index = require("./index");
+const fixtures = require("./fixtures/fixtures");
+const asana = require("asana");
+
+describe("Processes a local CSV correctly", () => {
+    // setup
+    const csv = fs.createReadStream("./fixtures/test.csv");
+    const parsedCsv = fixtures.parsedCsv;
+    jest.mock("asana");
+
+    test("Given a valid csv, loadCsv resolves with data", () => {
+        expect.assertions(1);
+
+        return index.loadCsv(csv).then(data => expect(data).toEqual(parsedCsv));
+    });
+
+    test("Given a parsed csv and a 30 day threshold, usersToDeprovision returns the correct users", () => {
+        expect.assertions(1);
+        const threshold = 30;
+        const promisedCsv = Promise.resolve(parsedCsv);
+        const result = ["guest1@external.com", "guest2@external.com"];
+
+        return index
+            .usersToDeprovision(promisedCsv, threshold)
+            .then(data => expect(data).toEqual(result));
+    });
+
+    test("Given a client, a domain, and an email address, deprovisionFunc calls `.removeUser`", () => {
```

Comment:
Perfect amount of testing and mocking. Not getting too into the weeds with what the library does, but any bug in the script could easily be written up as a new test following your pattern.